agent/util-scripts/pbench-tool-meister-start

```diff
@@ -27,23 +76,34 @@ import socket
 import sys
 import time
 
+from argparse import ArgumentParser
 from distutils.spawn import find_executable
 from pathlib import Path
 
 import redis
 
-import pbench.agent.toolmetadata as toolmetadata
-
+from pbench.agent import PbenchAgentConfig
+from pbench.agent.constants import (
+    redis_port as def_redis_port,
+    cli_tm_channel_prefix,
+    tm_channel_suffix_to_client,
+    tm_channel_suffix_from_client,
+    tm_channel_suffix_to_logging,
+)
+from pbench.agent.redis import RedisChannelSubscriber
 from pbench.agent.tool_data_sink import main as tds_main
 from pbench.agent.tool_meister import main as tm_main
```

Comment:
Consider renaming `main` in both of the above modules to something "else".

Reply:
This seems like a good use of "import as" functionality.  The "main" function of each of those modules on their own doesn't conflict.  The modules are named differently, so we could just `import pbench.agent.tool_data_sink`, and then then reference `pbench.agent.tool_data_sink.main` instead.  But that is long.  So these two uses of "as" are nice and convenient for where the code is used.